File: 0035-search-insert-position/solution.c
int searchInsert(int* nums, int numsSize, int target) {int count=0;
    for(int i=0;i<numsSize;i++,count++){
        if(nums[i]==target)
        return i;
        else if(target<nums[i])
        return i;
    }
    return count;
}

File: 0035-search-insert-position/solution.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        n = len(nums)
        lb = n
        low = 0
        high = n-1
        while low <= high:
            mid = (high - low)//2 + low
            if nums[mid] >= target:
                lb = mid
                high = mid - 1
            else:
                low = mid + 1
        return lb
